packages/react-card/Spec.md

```diff
@@ -0,0 +1,403 @@
+| Body of the card                                                                                                           |                      | CardBody          | ❌        |
+| Footer of the card                                                                                                         |                      | CardFooter        | ❌        |
+| Wrapper to render in column format                                                                                         |                      | CardColumn        | ❌        |
+| Allows for content to expand on hover                                                                                      |                      | CardExpandableBox | ❌        |
+
+...
+
+## API Proposal
+
+### Components
+
+| Component   | Purpose                                                                  |
+| ----------- | ------------------------------------------------------------------------ |
+| Card        | Wrapper for the main content of a card.                                  |
+| CardHeader  | Optional header slot for the card.                                       |
+| CardBody    | Main slot of the Card                                                    |
+| CardPreview | Previewing the document/article presented in the card through an image   |
+| CardFooter  | Optional footer slot for the card. (non-applicable for horizontal cards) |
+
+### Card
+
+#### Anatomy
+
+![visual anatomy of the Card component](./assets/Card.png)
+
+#### API
+
+| Property    | Values                                                                               | Default    | Purpose                                                                                   |
+| ----------- | ------------------------------------------------------------------------------------ | ---------- | ----------------------------------------------------------------------------------------- |
+| orientation | `vertical`, `horizontal`                                                             | `vertical` | Orientation of the card                                                                   |
+| size        | `smallest`, `smaller`, `small`, `medium`, `large`                                    | `medium`   | Define the minimum size of the card. Smaller sizes only apply to horizontal card          |
+| scale       | `fixed`, `auto-width`, `auto-height`, `auto`, `fluid-width`, `fluid-height`, `fluid` | `auto`     | Manages how the card handles it's scaling depending on the content                        |
+| onClick     | function                                                                             | undefined  | Function to call when clicking the card. Makes the card assume the interactive appearance |
+| appearance  | `filled`, `filled-alternative`, `outline`, `subtle`                                  | `filled`   | Define the appearance of the card                                                         |
```

Comment:
Not a complaint on this spec itself, but we should really define what the pattern for this is going forward as it is all over the place (i.e. one prop with defined values here vs a prop for each in `Button`).

Reply:
That's a good point. There's no previous defined pattern for this right? I only went with this approach as it seems to me to be much more manageable than several different props.
Are you aware of other components with appearance variations apart from `Button`?

Reply:
I think there are a couple other components that use it, but I don't remember which specific ones.

Reply:
I think this [RFC](19775) suggests using one prop with defined values.